app/src/docker/Dockerfile

```diff
@@ -3,6 +3,7 @@ ARG JAR_FILE
 ARG ENTRYPOINT_FILE
 WORKDIR /project
 COPY ${ENTRYPOINT_FILE} entrypoint.sh
+RUN chmod 777 /project/build/tomcat
 RUN chmod +x entrypoint.sh
```

Comment:
https://app.circleci.com/pipelines/github/department-of-veterans-affairs/abd-vro/934/workflows/15c1ccb2-c912-45e6-8858-6f15d35e75ea/jobs/847
```
FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:lintDockerfile'.
> src/docker/Dockerfile:7 DL3059 info: Multiple consecutive `RUN` instructions. Consider consolidation.
```